app/views/organizations/staff/pets/tabs/partials/_application_status.html.erb

```diff
@@ -1,16 +1,18 @@
 <div class="dropdown">
-  <button class="btn dropdown-toggle <%= application_status_classes(app)%>" data-bs-toggle="dropdown" aria-expanded="false">
+  <button class="btn dropdown-toggle <%= application_status_classes(app) %>" data-bs-toggle="dropdown" aria-expanded="false" <%= 'disabled' if app.status == 'adoption_made' %>>
     <%= app.human_enum_name(:status) %>
   </button>
-  <ul class="dropdown-menu">
-    <% AdopterApplication.statuses.keys.each do |status| %>
-      <%unless status == "adoption_made" %>
-        <li>
-          <%= button_to t("organizations.staff.pets.application_status.#{status}"),
-                        staff_adoption_application_review_path(app, params: { adopter_application: { status: status}}),
-                        :method => :patch, class: 'dropdown-item' %>
-        </li>
+  <% unless app.status == 'adoption_made' %>
```

Comment:
Do we need this check here? We already disable it above if the `app.status == 'adoption_made'` on page load, and when we create a new adoption [we redirect back](https://github.com/rubyforgood/pet-rescue/blob/34706588950d68476d5daac1e30d18464939a821/app/controllers/organizations/staff/matches_controller.rb#L5) which refreshes the page anyway...so that should be all we need?


Also, if you look at the [Create Adoption button](https://github.com/rubyforgood/pet-rescue/blob/34706588950d68476d5daac1e30d18464939a821/app/views/organizations/staff/pets/tabs/partials/_application_create_button.html.erb#L1), it is making a request with turbo stream. This is unnecessary I think. We can get away with an html request and it will redirect back. Do you agree?

Reply:
> Do we need this check here? We already disable it above if the `app.status == 'adoption_made'` on page load, and when we create a new adoption [we redirect back](https://github.com/rubyforgood/pet-rescue/blob/34706588950d68476d5daac1e30d18464939a821/app/controllers/organizations/staff/matches_controller.rb#L5) which refreshes the page anyway...so that should be all we need?
> 

Implementing this optimizes DOM composition by eliminating extraneous HTML to get render

Reply:
> Also, if you look at the [Create Adoption button](https://github.com/rubyforgood/pet-rescue/blob/34706588950d68476d5daac1e30d18464939a821/app/views/organizations/staff/pets/tabs/partials/_application_create_button.html.erb#L1), it is making a request with turbo stream. This is unnecessary I think. We can get away with an html request and it will redirect back. Do you agree?

Yes u are right, 
Done with the changes